libraries/shared/storage/storage_queue_test.go

```diff
@@ -7,27 +23,79 @@ import (
 
 	"github.com/vulcanize/vulcanizedb/libraries/shared/storage"
 	"github.com/vulcanize/vulcanizedb/libraries/shared/storage/utils"
+	"github.com/vulcanize/vulcanizedb/pkg/datastore/postgres"
 	"github.com/vulcanize/vulcanizedb/test_config"
 )
 
 var _ = Describe("Storage queue", func() {
-	It("adds a storage row to the db", func() {
-		row := utils.StorageDiffRow{
+	var (
+		db    *postgres.DB
+		row   utils.StorageDiffRow
+		queue storage.IStorageQueue
+	)
+
+	BeforeEach(func() {
+		row = utils.StorageDiffRow{
 			Contract:     common.HexToAddress("0x123456"),
 			BlockHash:    common.HexToHash("0x678901"),
 			BlockHeight:  987,
 			StorageKey:   common.HexToHash("0x654321"),
 			StorageValue: common.HexToHash("0x198765"),
```

Comment:
For future-proofing, does it make sense to see if it works without the leading `0x`?

Reply:
Or is it formatted that way for both geth and parity for example?

Reply:
@m0ar I think the `0x` is inconsequential when calling `common.HexToHash` since its [implementation](https://github.com/ethereum/go-ethereum/blob/97fb08342d227bbd126516083b0ddaf74e6e8468/common/types.go#L63) converts the string with [a function that chops off the `0x`](https://github.com/ethereum/go-ethereum/blob/97fb08342d227bbd126516083b0ddaf74e6e8468/common/bytes.go#L47).

Reply:
Yeah but I mean the storage diff code, does it always assume there is no leading `0x`?

Reply:
Nah the [code that converts the raw diffs to their internal representation](https://github.com/vulcanize/vulcanizedb/blob/staging/libraries/shared/storage/utils/row.go#L43) uses the same functions, so it'll handle hex values with and without the prefix.